.github/workflows/build.yml

```diff
@@ -23,7 +23,11 @@ jobs:
           DBNAME: testdb
           SAMPLEDB: true
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v2
+      with:
+        java-version: 8
+        distribution: 'zulu'
```

Comment:
Does this mean Zulu build? Any reason to choose this? https://jp.azul.com/downloads/zulu-community/

Reply:
Currently, `setup-java@v2` is just supporting two distributions `zulu & adopt` https://github.com/actions/setup-java

And `setup-java@v1` use `zulu` as default, so i keep it same as before

Reply:
Ah, got it. `v1` was Zulu... Thanks!